jaxrs/src/main/java/com/facebook/airlift/jaxrs/JaxrsBinder.java

```diff
@@ -43,8 +49,29 @@ public void bind(Key<?> targetKey)
         resourceBinder.addBinding().to(targetKey).in(SINGLETON);
     }
 
-    public void bindInstance(Object instance)
+    public ResourceBinding bindInstance(Object instance)
     {
         resourceBinder.addBinding().toInstance(instance);
+        return new ResourceBinding(instance.getClass(), binder);
+    }
+
+    public static class ResourceBinding
```

Comment:
Should this have a `public` modifier ?

Reply:
To be consistent with existing code 🙂 e.g. [HttpServerBinder](https://github.com/prestodb/airlift/blob/master/http-server/src/main/java/com/facebook/airlift/http/server/HttpServerBinder.java)